<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom"><channel><title>DNS on ЛюбимРешать</title><link>https://jiejue.ai/ru/dns/</link><description>Recent content in DNS on ЛюбимРешать</description><generator>Hugo</generator><language>ru-RU</language><lastBuildDate>Wed, 13 Aug 2025 15:03:02 +0000</lastBuildDate><atom:link href="https://jiejue.ai/ru/dns/index.xml" rel="self" type="application/rss+xml"/><item><title>Устранение неполадок с DNS в Kubernetes: когда поды не могут получить доступ к внешним доменам</title><link>https://jiejue.ai/ru/2025/02/k8s-dns-troubleshooting/</link><pubDate>Sat, 01 Feb 2025 08:50:37 +0400</pubDate><guid>https://jiejue.ai/ru/2025/02/k8s-dns-troubleshooting/</guid><description>&lt;p>В обычный будний день команда разработчиков столкнулась со странной проблемой: одна из подсистем в кластере Kubernetes вдруг не смогла получить доступ к внешнему домену. Казалось бы, это простая проблема, но она затрагивает Kubernetes DNS, Azure Private DNS и обнаружение служб. Давайте рассмотрим, как диагностировать и решить эту проблему.&lt;/p>
&lt;p>&amp;lt;! &amp;ndash;подробнее&amp;ndash;&amp;gt;&lt;/p>
&lt;h2 id="проблемное-явление">Проблемное явление&lt;/h2>
&lt;p>Разработчик обнаружил, что при попытке получить доступ к внешнему доменному имени из Pod он сталкивается с ошибкой разрешения домена:&lt;/p></description></item></channel></rss>